docs/concepts/submit-data.md

```diff
@@ -0,0 +1,221 @@
+# Submitting Data Blobs to Celestia
+
+To submit data to celestia, users submit blob transactions (`BlobTx`). Blob
+transactions contain two components, a standard cosmos-sdk transaction called
+`MsgPayForBlobs` and a Blob of data.
```

Comment:
```suggestion
`MsgPayForBlobs` and a blob of data.
```

Reply:
we don't capitalize this elsewhere [in our specs](https://github.com/celestiaorg/celestia-app/blob/main/x/blob/README.md), happy to defer to the review here, but wanted to bring that into consideration before accepting the recommendation

Reply:
maybe this works well to match [this](https://github.com/celestiaorg/celestia-app/blob/63915bf4cf32772fe75edf3e99df48315007aec6/x/blob/README.md?plain=1#L6)?